Plan/api/src/main/java/com/djrapitops/plan/delivery/web/resolver/request/Request.java

```diff
@@ -125,6 +161,7 @@ public String toString() {
                 ", query=" + query +
                 ", user=" + user +
                 ", headers=" + headers +
+                ", body=" + body +
```

Comment:
Better not have request body in toString if the goal is not allowing passwords from request body to be seen. `body.length` might be appropriate
```suggestion
                ", body=" + body.length +
```

Reply:
Totally missed that, thanks